src/nunavut/lang/__init__.py

```diff
@@ -673,7 +714,7 @@ def get_output_extension(self) -> str:
             return self._target_language.extension
         else:
             raise RuntimeError(
```

Comment:
It would benefit Nunavut to use more structured exception types. For the CLI use case it hardly matters but when integrating Nunavut as a library this becomes important. This is probably not related to this specific changeset but maybe it should be looked at in the future.

Reply:
Can you open an issue? I suppose you want exceptions that derive from some sort of "nunavut exception"?

Reply:
https://github.com/UAVCAN/nunavut/issues/224